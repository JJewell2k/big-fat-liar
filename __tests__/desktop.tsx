import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import App from '../pages/_components/Desktop/index'

describe('Desktop component', () => {
  let app: ShallowWrapper

  beforeEach(() => {
    app = shallow(<App />)
  })

  it('Should renders properly', () => {
    expect(shallowToJson(app)).toMatchSnapshot()
  })
})
