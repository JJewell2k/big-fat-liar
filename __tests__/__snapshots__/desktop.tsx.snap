// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Desktop component Should renders properly 1`] = `
<div
  className="flex flex-col items-end fixed left-0 top-0 h-screen w-screen border-box pt-6"
  onClick={[Function]}
>
  <button
    className="w-32 flex flex-col items-center my-2 p-1"
    data-testid="desktop-item-1"
    key="1"
    onClick={[Function]}
  >
    <div
      className="image-wrapper rounded-md border-2 border-transparent border-box px-1 py-2 bg-opacity-20"
    >
      <img
        alt="select-none dock item logo"
        className="w-12"
        src="/images/disk-icon.png"
      />
    </div>
    <h3
      className="select-none text-xs text-white font-bold whitespace-nowrap text-center mt-0.5 text-shadow-sm pl-2 pr-2 pt-0.5 pb-0.5 rounded-md"
    >
      Macintosh HD
    </h3>
  </button>
  <button
    className="w-32 flex flex-col items-center my-2 p-1"
    data-testid="desktop-item-2"
    key="2"
    onClick={[Function]}
  >
    <div
      className="image-wrapper rounded-md border-2 border-transparent border-box px-1 py-2 bg-opacity-20"
    >
      <img
        alt="select-none dock item logo"
        className="w-12"
        src="/images/folder-icon.png"
      />
    </div>
    <h3
      className="select-none text-xs text-white font-bold whitespace-nowrap text-center mt-0.5 text-shadow-sm pl-2 pr-2 pt-0.5 pb-0.5 rounded-md"
    >
      Giant Sur
    </h3>
  </button>
  <button
    className="w-32 flex flex-col items-center my-2 p-1"
    data-testid="desktop-item-3"
    key="3"
    onClick={[Function]}
  >
    <div
      className="image-wrapper rounded-md border-2 border-transparent border-box px-1 py-2 bg-opacity-20"
    >
      <img
        alt="select-none dock item logo"
        className="w-12"
        src="/images/pdf-icon.png"
      />
    </div>
    <h3
      className="select-none text-xs text-white font-bold whitespace-nowrap text-center mt-0.5 text-shadow-sm pl-2 pr-2 pt-0.5 pb-0.5 rounded-md"
    >
      Example.pdf
    </h3>
  </button>
</div>
`;
